src/Microsoft.Azure.Batch.SoftwareEntitlement.Server/Microsoft.Azure.Batch.SoftwareEntitlement.Server.csproj

```diff
@@ -21,11 +21,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Content Include="wwwroot\index.html">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </Content>
-  </ItemGroup>
```

Comment:
Ah, I didn't realize that the file is automatically included in the project, but it's not copied to the output directory by default. I've changed the `Include` directive to `Update`.